docs/documentation/upgrading/topics/keycloak/changes-22_0_0.adoc

```diff
@@ -129,4 +129,12 @@ The new artifact with Java EE support is `org.keycloak:keycloak-admin-client-jee
     <artifactId>keycloak-admin-client-jee</artifactId>
     <version>22.0.0</version>
 </dependency>
-----
\ No newline at end of file
+----
+
+= Passthrough proxy mode changes
+
+Keycloak's proxy configuration setting for mode *passthrough* does now longer parse HTTP forwarding headers in the request, as when a proxy forwards an HTTPS connection in passthrough mode, a proxy is unable to add, remove or update HTTP headers.
```

Comment:
@ahus1 Should be like this, right?

```suggestion
Keycloak's proxy configuration setting for mode *passthrough* no longer parses HTTP forwarding headers in the request, as when a proxy forwards an HTTPS connection in passthrough mode, a proxy is unable to add, remove or update HTTP headers.
```

Reply:
Or just `no longer parses`?

Reply:
Fixed in latest commit, ready for review.